test/common/ops/buy/buy.js

```diff
@@ -111,7 +111,7 @@ describe('shared.ops.buy', () => {
       params: {
         key: 'thankyou',
       },
-      type: 'special',
```

Comment:
is this a breaking api change? is there a way to avoid it?

Reply:
if we want to keep the `special` as type then I could revert this part, I just thought it would make it a bit less confusing for the developers and api users

Reply:
yeah, it's something we can consider for a new api version but for now we shouldn't change it